sycl/include/CL/sycl/detail/cg.hpp

```diff
@@ -353,7 +363,13 @@ class CG {
   // group to be executed.
   vector_class<Requirement *> MRequirements;
   // List of events that order the execution of this CG
-  vector_class<detail::EventImplPtr> MEvents;
+  std::vector<detail::EventImplPtr> MEvents;
+  // Member variables to capture the user code-location
+  // information from Q.submit(), Q.parallel_for() etc
+  std::string MFunctionName, 
+              MFileName;
+  int         MLine, 
+              MColumn;
```

Comment:
Have you applied clang-format to your changes? This doesn't look like properly formatted.

Reply:
That is possibly years of habit creeping in, but I can address that if it is important.

Reply:
@tovinkere it is a common practice in LLVM to use clang-format tool to take care of code style.  It would be great if you did so.